lib/app/layouts/conversation_view/widgets/text_field/conversation_text_field.dart

```diff
@@ -823,11 +851,14 @@ class TextFieldComponent extends StatelessWidget {
       onKeyEvent: (_, ev) => handleKey(_, ev, context, isChatCreator),
       child: Padding(
         padding: const EdgeInsets.only(right: 5.0),
-        child: Container(
+        child: ValueListenableBuilder<bool>(
+        valueListenable: isRecordingNotifier,
+        builder: (context, isRecording, child) {
+        return Container(
           decoration: iOS
               ? BoxDecoration(
                   border: Border.fromBorderSide(BorderSide(
-                    color: context.theme.colorScheme.properSurface,
+                    color: (isRecording & iOS) ? CupertinoDynamicColor.resolve(iOSVoiceRecorderBackgroundColor, context) : context.theme.colorScheme.properSurface,
```

Comment:
Make this `context.theme.colorScheme.primary.withOpacity(0.3)` (can modify opacity so it matches iOS opacity best)